app/controllers/user.js

```diff
@@ -31,3 +31,26 @@ exports.signIn = (req, res, next) =>
       }
     })
     .catch(next);
+
+exports.getAllUsers = (req, res, next) => {
+  const limit = req.query.limit || 10;
+  const page = req.query.page || 1;
+  const paginationParams = {
+    limit,
+    page,
+    offset: (page - 1) * limit,
+    orderBy: req.query.orderBy || 'email',
+    order:
+      req.query.order && (req.query.order.toUpperCase() === 'ASC' || req.query.order.toUpperCase() === 'DESC')
+        ? req.query.order.toUpperCase()
+        : 'ASC'
+  };
+
+  return userService
+    .findAllPagination(paginationParams)
+    .then(users => {
+      users.totalPages = Math.ceil(users.count / limit);
```

Comment:
We usually don't modify objects passed in params, simply do something like `res.send({ ...users, totalPages: bla })` , could be returned directly by service